src/content/docs/en/reference/routing-reference.mdx

```diff
@@ -170,6 +170,39 @@ const { post } = Astro.props;
 <h1>{id}: {post.name}</h1>
 ```
 
+### `routePattern`
+
+<p>
+
+**Type:** `string` <br />
+<Since v="5.15.0" />
+</p>
+
+A property available in [`getStaticPaths()`](#getstaticpaths) options to access the current [`routePattern`](/en/reference/api-reference/#routepattern) as a string.
+
+This provides data from the [Astro render context](/en/reference/api-reference/) that would not otherwise be available within the scope of `getStaticPaths()` and can be useful to calculate the `params` and `props` for each page route.
+
+`routePattern` always reflects the original dynamic segment definition in the file path (e.g. `/[...locale]/[slug]`), unlike `params`, which are explicit values for a page (e.g. `/fr/post-1/`).
```

Comment:
I think we should change the route pattern to the same example `[...locale]/[files]/[slug]` like in the code block bellow.

Reply:
Thanks! OK, updated fichiers, but in your scenario, would someone therefore *also* want the slug to be `poste-1` do you think? @openscript

Reply:
(Or, if these are not "blog posts" at all, do we want something entirely different here?)